content/applications/finance/payment_acquirers/wire_transfer.rst

```diff
@@ -1,40 +1,46 @@
-===================================
-How to get paid with wire transfers
-===================================
+==============
+Wire transfers
+==============
 
-**Wire Transfer** is the default payment method available.
-The aim is providing your customers with your bank details
-so they can pay on their own.
-This is very easy to start with but slow and inefficient process-wise.
-Opt for payment acquirers as soon as you can!
+The **Wire transfer** payment method allows you to provide payment instructions to your customers,
+such as the bank details and the communication to be used. They are displayed:
 
-How to provide customers with payment instructions
-==================================================
+- at the end of the checkout process, once the customer has selected **Wire transfer** as a payment
+  method and clicked the :guilabel:`Pay now` button:
 
-Put your payment instructions in the **Thanks Message** of your payment method.
+  .. image:: wire_transfer/payment_instructions_checkout.png
+     :align: center
+     :alt: Payment instructions at checkout
 
-.. image:: wire_transfer/payment_instructions.png
-   :align: center
+- on the customer portal:
 
-They will appear to the customers when they place an order.
+  .. image:: wire_transfer/payment_instructions_portal.png
+     :align: center
+     :alt: Payment instructions on the customer portal
 
-.. image:: wire_transfer/payment_customer_instructions.png
-   :align: center
+.. note::
+
+   - While this method is very accessible and requires minimal setup, it is very inefficient
+     process-wise. We recommend setting up a :doc:`payment acquirer <../payment_acquirers>` instead.
+   - Online orders remain in the "Quotation sent" (i.e., unpaid order) stage until you receive
+     the payment and :guilabel:`Confirm` the order.
 
-How to manage an order once you get paid
-========================================
+.. tip::
+   **Wire transfer** can be used as a template for other payment methods that are processed
+   manually, such as checks. Just rename or duplicate it.
 
-Whenever a customer pays by wire transfer, the order stays in an
-intermediary stage **Quotation Sent** (i.e. unpaid order).
-When you get paid,
-you confirm the order manually to launch the delivery.
+Configuration
+=============
 
-How to create other manual payment methods
-==========================================
+**Wire Transfer** is installed and enabled by default.
 
-If you manage a B2B business, you can create other manually-processed
-payment methods like paying by check.
-To do so, just rename *Wire Transfer* or duplicate it.
+Select the :guilabel:`Communication` to be used in the :guilabel:`Configuration` tab:
```

Comment:
I don't think all readers know where to find Wire Transfer, I think you could add a path before this §
```suggestion
To configure the payment method, go to :menuselection:`Accounting --> Configuration --> Payments -->
Payment Acquirers`, and open the :guilabel:`Wire Transfer` card. Then, select the
:guilabel:`Communication` to be used in the :guilabel:`Configuration` tab:
```

Reply:
Also, "Payment Acquirers" disappears in 16.0 and is replaced by Payment Providers.